README.md

```diff
@@ -73,6 +73,7 @@ Tenere can be configured using a TOML configuration file. The file should be loc
 
 - Linux : `$HOME/.config/tenere/config.toml` or `$XDG_CONFIG_HOME/tenere/config.toml`
 - Mac : `$HOME/Library/Application Support/tenere/config.toml`
+- Windows: `~/AppData/Roaming/tenere/config.toml` *(Please create.)*
```

Comment:
No need for this as it says in the line above that the file should be located in that folder.

Reply:
But it wasn't.

Reply:
The folder as well didn't exist.

Reply:
Yes it does not exist. It is up to the user to create it. In that case this applies to all the OS. Maybe we can add a note that it is not created by default

Reply:
I'll then remove it